### ISSUE TITLE
학습 시나리오 3 테스트

### ISSUE THREAD

**Issue by TukBang** [TukBang/Pet-care-app#5]
모델 분류 기준:
1. 해상도가 학습 비율에 2배 이상 차이나거나 그에 가까운 수준 (확대)
 - 130x130 이하 -> 224 x 224
2. 해상도가 학습 비율에 2배 이상 차이나거나 그에 가까운 수준 (축소)
 - 392x392 이상 -> 224 x 224
3. 그 이외
 - 1, 2를 제외한 나머지 -> 224 x 224

Model 1: A1 ~ A6 (6-CLASS), EfficientNetV2_s (130x130 이하)
Model 2: A1 ~ A6 (6-CLASS), EfficientNetV2_s (392x392 이상)
Model 3: A1 ~ A6 (6-CLASS), EfficientNetV2_s (그 외)
Preprocessing: ratio-and-zero-padding, interpolation: LANCZOS4
batch_size: 128
optimizer = Adam
learning rate: 1e-4
criterion = CrossEntropyLoss

batch_size는 앞선 시나리오 1, 2에서 128이 학습속도가 더 빠르고 수치가 높아 128로만 진행하며
Model 1의 경우 해상도를 168x168로
Model 2, 3의 경우 해상도를 224x224로 진행

**Comment by TukBang** [comment]
[Model 1]
![graph](https://user-images.githubusercontent.com/121234327/231403902-1357b790-c7be-46f5-ac93-6d8aa5a04354.png)

[Model 2]
![graph](https://user-images.githubusercontent.com/121234327/231404113-53f8abda-87fa-464b-9368-4180033d46c5.png)

[Model 3]
![graph](https://user-images.githubusercontent.com/121234327/231404178-c0d84847-1cdc-4b54-aeff-598ecd4babc8.png)

해상도가 높을수록 정확도가 상승

**Comment by TukBang** [comment]
[Model 1]
![AUC-ROC](https://user-images.githubusercontent.com/121234327/231404352-57fd5425-64d6-4a7a-900e-f191f0a9d7fe.png)

[Model 2]
![AUC-ROC](https://user-images.githubusercontent.com/121234327/231404381-500570f6-c56c-4add-9ffe-d829d4ce597d.png)

[Model 3]
![AUC-ROC](https://user-images.githubusercontent.com/121234327/231404642-a61678dd-0b18-439b-bea0-e9032b2fb5aa.png)

**Comment by TukBang** [comment]
[Model 1]

              precision    recall  f1-score   support

          A1       0.46      0.57      0.51       442
          A2       0.49      0.57      0.53       280
          A3       0.37      0.42      0.39       273
          A4       0.56      0.68      0.61       669
          A5       0.74      0.32      0.44       503
          A6       0.72      0.57      0.64       141

    accuracy                           0.53      2308
    
[Model 2]

              precision    recall  f1-score   support

          A1       0.57      0.56      0.57       557
          A2       0.60      0.71      0.65       728
          A3       0.56      0.69      0.62       774
          A4       0.57      0.54      0.55       417
          A5       0.59      0.33      0.42       552
          A6       0.89      0.77      0.83       359

    accuracy                           0.61      3387
    
[Model 3]

              precision    recall  f1-score   support

          A1       0.60      0.45      0.52        88
          A2       0.81      0.55      0.65       323
          A3       0.60      0.84      0.70       270
          A4       0.00      0.00      0.00         6
          A5       0.00      0.00      0.00         1
          A6       0.93      0.88      0.90       262

    accuracy                           0.71       950

Accuracy만 봤을 때는 해상도가 높을 수록 높지만,
각 클래스마다 해상도 차이로 인한 예측 변동이 있어서
해상도가 높다고해서 절대적으로 성능이 좋아진다고 볼 수 없음
